iina/Utility.swift

```diff
@@ -13,6 +13,14 @@ class Utility {
   static let tabTitleFontAttributes = FontAttributes(font: .system, size: .system, align: .center).value
   static let tabTitleActiveFontAttributes = FontAttributes(font: .systemBold, size: .system, align: .center).value
 
+  static let supportedFileExt: [MPVTrack.TrackType: [String]] = {
+    var result: [MPVTrack.TrackType: [String]] = [:]
```

Comment:
Swift used to have performance issue when compiling complicated dict literals if I remember correctly. I'll switch to literal since it's not a problem nowadays.